bundles/ruby/init.moon

```diff
@@ -9,7 +9,8 @@ mode_reg =
     'rb',
     'rbw',
     'gemspec',
-    'builder'
+    'builder',
+    'cr'
```

Comment:
I'm going to have to say no to this one. While it seems to have a very similar syntax to Ruby, it's not Ruby (and while the syntax is similar, it's not the same). For now, unless you want to write a proper mode for it, I suggest something like this in the init file:

```moonscript
howl.mode.register {
  name: 'crystal',
  extensions: 'cr',
  parent: 'ruby',
  create: -> {}
}
``` 

This will create an empty Crystal mode that is derived from the Ruby mode, meaning lexing, etc., will work the same way.